php/libraries/NDB_Config.class.inc

```diff
@@ -447,7 +447,13 @@ class NDB_Config
         // nothing in the config file, so get the value from the DB
         // This will throw a ConfigurationException if it does not
         // exist.
-        return $this->getSettingFromDB($name);
+        try {
+            return $this->getSettingFromDB($name);
+        } catch (ConfigurationException $e) {
+            error_log($e);
```

Comment:
this will trigger an array tp string conversion. And I don't think it is useful to fill the log with that. The calling class should decide what to do.